packages/console/src/monaco/MonacoUtils.ts

```diff
@@ -339,91 +340,60 @@ class MonacoUtils {
     return platform.startsWith('Mac');
   }
 
+  // Tracks whether removeConflictingKeybindings() has been called
+  static conflictingKeybindingsRemoved = false;
+
   /**
    * Remove any keybindings which are used for our own shortcuts.
-   * This allows the key events to bubble up so a component higher up can capture them
-   * @param editor The editor to remove the keybindings from
+   * This allows the key events to bubble up so a component higher up can capture
+   * them. Note that this is a global configuration, so all editor instances will
+   * be impacted.
    */
-  static removeConflictingKeybindings(
-    editor: monaco.editor.IStandaloneCodeEditor
-  ): void {
-    // Multi-mod key events have a specific order
-    // E.g. ctrl+alt+UpArrow is not found, but alt+ctrl+UpArrow is found
-    // meta is WindowsKey on Windows and cmd on Mac
-    // ctrl is ctrl Windows and ctrl on Mac
-    // alt is alt on Windows and option on Mac
-    const keybindings = [
+  static removeConflictingKeybindings(): void {
+    // This function adds the keybindings to the global registry, so we only
+    // want to do it once
+    if (MonacoUtils.conflictingKeybindingsRemoved) {
+      return;
+    }
+
+    MonacoUtils.conflictingKeybindingsRemoved = true;
+
+    /* eslint-disable no-bitwise */
+    monaco.editor.addKeybindingRules([
+      // Restart console in Enterprise (see Shortcuts.ts)
       {
-        windows: 'ctrl+D',
-        mac: 'meta+D',
+        keybinding: monaco.KeyMod.CtrlCmd | monaco.KeyCode.KeyD,
+        command: null,
       },
+      // Console focus history in Community (see ConsoleShortcuts.ts)
       {
-        windows: 'ctrl+H',
+        keybinding: monaco.KeyMod.WinCtrl | monaco.KeyCode.KeyH,
```

Comment:
This isn't working correctly on Windows/Linux. You need to remove `monaco.KeyMod.WinCtrl | monaco.KeyCode.KeyH` on Mac, but `monaco.KeyMod.CtrlCmd | monaco.KeyCode.KeyH` on Windows/Linux. We use Ctrl+H on Windows and Ctrl+Option+H for this shortcut on mac:
```
    shortcut: [MODIFIER.CTRL, KEY.H],
    macShortcut: [MODIFIER.CTRL, MODIFIER.OPTION, KEY.H], // cmd+H is a system level shortcut on Mac, so use something else
```

With this change pressing Ctrl+H while in the console input brings up the Find/Replace widget:
![image](https://github.com/deephaven/web-client-ui/assets/4505624/d6cc8646-49f3-468d-ac2e-37a4b88f8b8b)

Granted this keyboard shortcut is kind of janky already, since it only works if the Command History panel is visible, but may as well not make it worse.

Reply:
To clarify, I was under the impression that WinCtrl + KeyH wasn't a thing on Mac so no need to remove it? It didn't appear to be handled at all for the Mac case in the original code

Reply:
WinCtrl is windows key (meta key) on windows/Linux and Control on Mac

CtrlCmd is control on windows/Linux and cmd on Mac

Reply:
Ah, I misinterpreted "WinCtrl" to be "ctrl on Windows"

Reply:
Naming stuff is hard :stuck_out_tongue: